src/client/components/Footer/Footer.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import logo from '*../../../src/client/assets/images/logo.png';
+import send from '*../../../src/client/assets/images/emailsend.png';
+import './Footer.css';
+import { Link } from 'react-router-dom';
+
+function Footer() {
+  return (
+    <div>
+      <footer>
+        <ul>
+          <li className="footer-list">
```

Comment:
At least the logo must be extracted into its own component to be reused elsewhere. Each of the links are separate components also